docs/MergingPrs.md

```diff
@@ -15,16 +15,18 @@ Merging to `master` or `release` branches requires approval from both _main revi
 ### How to merge
 
 1. Make sure that GitHub gives the green light for merging.
-  There are a few scenarios where GitHub can prevent merging from proceeding:
-  * **Merge conflict**: The PR is conflicting with the current `master` branch; the author will
-    need to resolve the conflicts before proceeding. 
-  * **Check failure**: E.g. CI is failing.
-  * **Outdated branch**: The PR is not in sync with the current `master` branch; the author will
-    need to sync it before proceeding. This can be done via GitHub with the "Update branch" button.
+  There are a few scenarios where GitHub can prevent merging. In those cases, the dev should resolve the
+  problem as given below and submit a new iteration.
+  1. **Merge conflict**: The PR is conflicting with the current `master` branch. The author should rebase the branch
+    and resolve conflicts.
+  1. **Check failure**: E.g. CI is failing. The author should fix the causes of the check failures.
```

Comment:
It's best to require dev to either fix or justify _all_ check failures. I'll updated accordingly.